scripts/eslint-plugin/rules/no-global-node.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+const errorMessage = 'Avoid referencing the global Node constructor.';
+
+function isGlobalRef(reference) {
+    return reference.resolved === null || reference.resolved.scope.type === 'global';
+}
+
+module.exports = {
+    meta: {
+        type: 'problem',
+        docs: {
+            description: errorMessage,
+        },
+    },
+
+    create(context) {
+        return {
+            'Identifier[name="Node"]': function (node) {
```

Comment:
It is strange to only cache `Node`. What about other APIs like `EventTarget` or `document`?